meta-resin-common/recipes-connectivity/networkmanager/files/99dhcp_ntp

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+added_servers_file=/tmp/chrony_added_dhcp_ntp_servers
+
+case "$2" in
+	dhcp4-change|up|connectivity-change)
+	if [[ -n $DHCP4_NTP_SERVERS ]]; then
+    # Tell chronyd to use this server
+		/usr/bin/chronyc add server $DHCP4_NTP_SERVERS
+    # Save the ntp server we received and interface information to a file
+    echo "resin_$1="""$DHCP4_NTP_SERVERS"" >> $added_servers_file
```

Comment:
It would be nice to use a descriptive variable for $1 (INTERFACE?).

Reply:
Yes $1 is the interface. I'll change it